canvas_site_creator/api.py

```diff
@@ -354,8 +354,7 @@ def course_instances(request, sis_term_id, sis_account_id):
 
         # fixes TLT-1570 where courses that already have a Canvas course were showing up
         # in the create list
-        query_set = query_set.exclude(canvas_course_id__isnull=False)
-
+        query_set = query_set.exclude(sync_to_canvas=1)
```

Comment:
I wonder if there would be a case where `sync_to_canvas=0` but `canvas_course_id` is not `null`, e.g. because of some other edge case we can't imagine. Would it be worth ensuring both possibilities are `exclude`d? i.e. ensure it matches the logic in the [`create_job` view](https://github.com/Harvard-University-iCommons/canvas_account_admin_tools/pull/317/files#diff-e04e401071cebc10a9425bd6d0e47345d1c6b880fc7dadb37596edcd7e4de409R323-R324)?

Reply:
We might also consider making the code comment here more generic, and not referencing that older ticket anymore.